src/test/java/stepsdefinition/Hooks.java

```diff
@@ -1,22 +1,33 @@
 package stepsdefinition;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
 import com.cucumber.listener.Reporter;
 
 import cucumber.api.java.After;
 import cucumber.api.java.Before;
+import graph.LocustBarChart;
 
 public class Hooks {
 	
 	 @Before
 	 public void BeforeSteps() {
+		 	//Setup the Report basic information
 	        Reporter.setSystemInfo("User Name", System.getProperty("user.name"));
 	        Reporter.setSystemInfo("Time Zone", System.getProperty("user.timezone"));
 	        Reporter.setSystemInfo("Machine", System.getProperty("os.name"));
 	    }
 	 
 	 @After(order = 0)
-	 public void AfterSteps() {
-	    
+	 public void AfterSteps() throws IOException {
+		 //Create the chart after Scenario iteration
+		 LocustBarChart locustBarChart = new LocustBarChart();
+	     locustBarChart.createChart(0);
+		 File destinationPath = new File(Paths.get("target/cucumber-reports/locustcharts/" + locustBarChart.getFileName()).toFile().getAbsolutePath());
```

Comment:
You are handling directories in too many places! Why not set the base working directory from here, and then all the other path based on this?

Reply:
Yes man! I know it! This could be another task for the kanban board. Try to make a config file or something like that where store this kind of information in order to not duplicate it or to have it in different places. Maybe we could discuss this in the stand up.

Reply:
Since you are adding a new file location here, lets make the change in this PR.

Reply:
Ok. I will take a look and let's see how this can be done in our approuch ...